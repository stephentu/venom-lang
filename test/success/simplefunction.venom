x = 1;
def f(x::int) -> int = x + 1; end
x = x + 2;
y = f(x);
print ("hello, world");
print(x);
print(f(x));
