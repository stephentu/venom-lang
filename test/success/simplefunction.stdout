hello, world
3
4
